### ISSUE TITLE
Color problems with "Hero, overlapped book cover with links"

### ISSUE THREAD

**Issue by carolinan** [WordPress/twentytwentyfive#473]
**Description**

In some of the style variations, the text color inside the cover block is not visible enough.

**Step-by-step reproduction instructions**
Select the "Evening" style variation.
Insert a "Hero, overlapped book cover with links" pattern.
Confirm that the text has a color contrast that is too low against the background:

Repeat for Twilight, Sunrise, and Midnight.

https://github.com/user-attachments/assets/17a38209-f807-445b-be89-e03cd1dfa9e3

**Comment by beafialho** [comment]
I'm not sure what design feedback is needed here. This seems like a technical bug.

**Comment by juanfra** [comment]
Sorry, I didn't add the context.

What's happening here is that the cover block for the pattern is using a custom gradient as the background. That means that it works really well for the default style variation, but at the same time it'll stay like that for all of the other variations. That's because the gradient is not using a variable, it is just a custom gradient (`linear-gradient(180deg,rgb(251,250,243) 51%,rgb(255,255,255) 51%)`)

The things that come to mind to resolve this issue:

- Try a total different approach for implementing this pattern where we don't use a custom gradient.
- Add a gradient to all style variations and use that one. If we do that, we'll need design feedback. And we'll need designs.
- Leave the custom gradient, and use a hardcoded text color.

Maybe there's some other option I may be missing.

**Comment by beafialho** [comment]
Can we use a variable background color that works well for all of them?

**Comment by juanfra** [comment]
@beafialho I'm afraid that if we use a variable background color, then we won't have the "split" background anymore. And it will be a plain background for the whole pattern (unless we try a different implementation, which I don't know if it's even possible). 

If we go that route, then we can rely on "Style 1".

**Comment by beafialho** [comment]
Also having a hard time providing useful input here, as this surpasses my technical knowledge. @jasmussen pinging in case you have suggestions?

**Comment by jasmussen** [comment]
Tricky one. It sounds like this is the core of the issue:

> What's happening here is that the cover block for the pattern is using a custom gradient as the background. That means that it works really well for the default style variation, but at the same time it'll stay like that for all of the other variations. That's because the gradient is not using a variable, it is just a custom gradient (linear-gradient(180deg,rgb(251,250,243) 51%,rgb(255,255,255) 51%))

It seems there are two ways forward:

1. Find a way to build that pattern, or a version of the pattern that keeps the spirit of it, but which does _not_ use the linear gradient trick for the background, so it can rely on section styles. Not sure if making the book "sticky" could create that kind of overlap? 
2. Keep the linear-gradient, but then bake in the text color, at least in the area on top of the light yellow color, so the text color does not change when switching style variations. 

For 1, beyond any better ideas that would let us create the split look, two quick tests would be:

![Image](https://github.com/user-attachments/assets/497d15fc-6404-4f49-8aa7-ea1038826d43)

![Image](https://github.com/user-attachments/assets/4bfc293e-e4cc-4bee-a520-e7e3526e290f)

They lose a bit of character, but perhaps that's preferable so they can keep the section style magic?

**Comment by beafialho** [comment]
Thanks for chiming in @jasmussen.

Whatever path you're all most comfortable with. Obviously this would be a cool layout to include, but if we end up removing the split background, I would just reposition the elements on the left so that there isn't too much spacing between them:

![Image](https://github.com/user-attachments/assets/39bc8fec-2939-4518-93ab-c1b7e83f7a8b)

**Comment by juanfra** [comment]
I've been looking at this one and couldn't find an elegant way to keep the design as is with a split background. I believe at this point, even with the loss of character, it is preferable to have it working with the section styles.
